testing/simapp/app.go

```diff
@@ -613,6 +613,10 @@ func NewSimApp(
 	app.BasicModuleManager.RegisterLegacyAminoCodec(legacyAmino)
 	app.BasicModuleManager.RegisterInterfaces(interfaceRegistry)
 
+	// NOTE: upgrade module is required to be prioritized
+	app.ModuleManager.SetOrderPreBlockers(
+		upgradetypes.ModuleName,
+	)
```

Comment:
Nice one! I think we can remove `upgradetypes.ModuleName` from `SetOrderBeginBlockers` below now

Reply:
addressed in https://github.com/cosmos/ibc-go/pull/4676/commits/b8d8893e17eab0366ba5232295e0459933db61da